src/models/recipe/recipe.schema.ts

```diff
@@ -0,0 +1,30 @@
+import { Schema, model } from "mongoose";
+import mongoose from "mongoose";
+import { IRecipeDocument, IRecipeModel } from "./recipe.types";
+
+//Recipe Schema
+const RecipeSchema: Schema = new Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  desription: {
+    type: String,
+    required: true,
+  },
+  likes: {
+    key: String,
+    ref: "user",
+    value: Date,
+  },
+  postedBy: {
+    type: String,
+    ref: "user",
+  },
+  images: {
+    type: [{ url: String }],
+  },
+});
```

Comment:
Don't forget to add a timestamp property to this Schema so mongo can auto-keep track of createAt and updatedAt attributes. https://mongoosejs.com/docs/guide.html#timestamps